lib/cluster-providers/generic-cluster-provider.ts

```diff
@@ -236,18 +240,26 @@ export class GenericClusterProvider implements ClusterProvider {
     createCluster(scope: Construct, vpc: ec2.IVpc, secretsEncryptionKey: IKey | undefined): ClusterInfo {
         const id = scope.node.id;
 
+        const builder = scope as unknown as BlueprintBuilder;
```

Comment:
You're right.  I'm not sure how to figure out whether or not the version function has been called, since the properties do not get passed onto the blueprint.